src/cameras/search/search-input.jsx

```diff
@@ -1,7 +1,15 @@
 import React from 'react'
+import { Grid } from '@material-ui/core'
 import SearchBar from 'material-ui-search-bar'
 
+import { SearchButton } from 'cameras/search/search-button'
 import { useInputProps } from 'cameras/search/hooks/use-input-props'
+import { useContext } from 'cameras/context'
 
-export const SearchInput = () => <SearchBar {...useInputProps()} />
+export const SearchInput = () => <Grid item xs={12} sm={12} md={12} lg={12}>
+  <SearchBar {...useInputProps(useContext().setSearchInput)} />
```

Comment:
No need to pass in the `setSearchInput` parameter here, since it coming from a context (and thus being accessible via calling a hook) allows us to do this:
![image](https://user-images.githubusercontent.com/23569938/129751885-9133542f-6d65-4bf1-8f2c-324c0afe862c.png)

Reply:
Done!

Reply:
![image](https://user-images.githubusercontent.com/23569938/129866556-9afe18d7-c447-4923-8ac8-60adc542af1e.png)
Did we forget to also remove the prop from here? :smile:

Reply:
Done!